pages/configs/build-cache_json.md

```diff
@@ -78,9 +78,9 @@ generates for **build-cache.json**:
     // "s3Region": "us-east-1",
 
     /**
-     * The name of the bucket in Amazon S3 to use for build cache.
+     * (Required) The name of the bucket in Amazon S3 to use for build cache.
      */
-    // (Required) "s3Bucket": "my-bucket",
+    // "s3Bucket": "my-bucket",
```

Comment:
@faithfinder thanks for making this PR!  We overlooked it because the Rush website has been moved to a new monorepo for Docusaurus.  It looks like this issue is now fixed in that repo.

Reply:
I wonder how [that](https://github.com/microsoft/rushstack-websites/pull/70) happened :-P